src/d/alexa-openwebif/source/main.d

```diff
@@ -1,6 +1,7 @@
 import std.stdio;
 import std.xml;
 import std.string;
+import std.algorithm.searching;
```

Comment:
please use local import when u need this module just in your new method and use selective importing of just the symbol used `import std.algorithm.searching:countUntil;`